src/liblog/lib.rs

```diff
@@ -445,16 +444,14 @@ mod tests {
 
     #[test]
     fn match_full_path() {
-        let dirs = [
-            LogDirective {
-                name: Some("crate2".to_string()),
-                level: 3
-            },
-            LogDirective {
-                name: Some("crate1::mod1".to_string()),
-                level: 2
-            }
-        ];
+        let dirs = [LogDirective {
+            name: Some("crate2".to_string()),
+            level: 3,
+        },
+                    LogDirective {
```

Comment:
Yeah, this is broken. But I think a fix has already landed in Rustfmt. @goyox86, could you try re-running and see if this gets fixed?

https://github.com/nrc/rustfmt/pull/449